packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/tags/GetTagsTests.java

```diff
@@ -72,6 +77,130 @@ public void getTagsWithConsumerRole()
             .and().entriesListContains("tag", documentTagValue2.toLowerCase());
     }
 
+    /**
+     * Include count in the query parameters and ensure count is as expected for returned tags.
+     */
+    @Test(groups = { TestGroup.REST_API, TestGroup.TAGS, TestGroup.REGRESSION })
+    public void testGetTags_withIncludeCount()
+    {
+        STEP("Get tags including count filter and ensure count is as expected for returned tags");
+        returnedCollection = restClient.authenticateUser(adminUserModel)
+                .withParams("include=count")
+                .withCoreAPI()
+                .getTags();
+        restClient.assertStatusCodeIs(HttpStatus.OK);
+
+        returnedCollection.getEntries().stream()
+                .filter(e -> e.onModel().getTag().equals(folderTagValue) || e.onModel().getTag().equals(documentTagValue))
+                .forEach(e -> e.onModel().assertThat().field("count").is(2));
+
+        returnedCollection.getEntries().stream()
+                .filter(e -> e.onModel().getTag().equals(documentTagValue2))
+                .forEach(e -> e.onModel().assertThat().field("count").is(1));
+    }
+
+    /**
+     * Get tags and order results by count. Default sort order should be ascending
+     */
+    @Test(groups = { TestGroup.REST_API, TestGroup.TAGS, TestGroup.REGRESSION })
+    public void testGetTags_withOrderByCountDefaultOrderShouldBeAsc()
+    {
+
+        STEP("Get tags and order results by count. Default sort order should be ascending");
+        returnedCollection = restClient.authenticateUser(adminUserModel)
+                .withParams("include=count&orderBy=count")
```

Comment:
Out of interest what happens if count is not included, but we request to sort by it? I'm not sure if we need a test for this, but we should document the behaviour in the API explorer.

Reply:
The way I wrote the code, we can only sort by count when count is also included. That being said, do you think that when count isn't included and we request to sort by it, we should let the user know why it isn't possible?

Reply:
That would be ideal. This is an edge case, but it would be good to document the behaviour and provide a helpful error message.